content/pages/employment/job-seekers/less-than-one-year.md

```diff
@@ -39,7 +39,7 @@ relatedlinks:
 ##### 12 Months Before Leaving Active Duty
 
 - Fill in any gaps in your education or skill set by taking classes and obtaining certifications and licensures. The [GI Bill](/education/gi-bill/) may be used for some of this prior to exiting the service.
-- Contact your local Transition Assistance Program (TAP) manager to schedule the mandatory Transition Goals, Plans, Success (GPS) curriculum. If you are unable to attend this training, it is available through [Joint Knowledge Online](https://jkodirect.jten.mil), but you must have approval from your service branch to get credit. You must complete the VA Benefits Briefings I and II, the Department of Labor Employment Workshop, and an Individual Transition Plan (ITP). ITPs are service specific, so [check documents related to your branch](https://dodtap.mil/my.policy).
+- Contact your local Transition Assistance Program (TAP) manager to schedule the mandatory Transition Goals, Plans, Success (GPS) curriculum. If you are unable to attend this training, it is available through [Joint Knowledge Online](http://jko.jten.mil/), but you must have approval from your service branch to get credit. You must complete the VA Benefits Briefings I and II, the Department of Labor Employment Workshop, and an Individual Transition Plan (ITP). ITPs are service specific, so [check documents related to your branch](https://dodtap.mil/my.policy).
```

Comment:
Okay, probably no httpS for this domain

Reply:
http://jko.jten.mil/videos/JKO_FOGO_Video_032015_final.mp4